lib/ui/screens/filter_screen.dart

```diff
@@ -0,0 +1,122 @@
+                          subtitle: Text(_getOrganizerFilterString(model.organisers, model.showOrganizer)),
+                          trailing: const Icon(Icons.keyboard_arrow_right),
+                          onTap: () async {
+                            final result = await Navigator.of(context)
+                                .push(MaterialPageRoute(
+                              builder: (context) => OrganiserFilterScreen(
+                                  organisers: model.organisers,
+                                  showOrganizer: model.showOrganizer),
+                            ));
+
+                            if (result != null) {
+                              model.setShowOrganizer(
+                                  result as Map<String, bool>);
+                            }
+                          },
+                        ),
+                        ListTile(
+                          title: const Text("Gremiensitzungen ausblenden"),
+                          trailing: Switch(
+                            materialTapTargetSize:
+                            MaterialTapTargetSize.shrinkWrap,
+                            onChanged: (value) {
+                              model.setHideGremien(value);
+                            },
+                            value: model.hideGremien,
+                          ),
+                          onTap: () {
+                            model.setHideGremien(!model.hideGremien);
+                          },
+                        ),
+                        ListTile(
+                          title: const Text("Nach Datum filtern"),
+                          subtitle: model.dateTimeRange != null
+                              ? Text(model.dateTimeRange!.startEndString())
+                              : null,
+                          trailing: const Icon(Icons.date_range),
+                          onTap: () async {
+                            final result = await showDateRangePicker(
+                              context: context,
+                              firstDate: DateTime.now(),
+                              lastDate:
+                                  DateTime.now().add(const Duration(days: 365 * 2)),
+                              initialDateRange: model.dateTimeRange,
+                            );
+
+                            model.setDateTimeRange(result);
+                          },
+                        ),
+                        ElevatedButton(onPressed: (){
+                          model.resetFilterSettings();
+                        }, child: const Text("Filter zurücksetzen"))
+                      ],
+                    ),
+                  ),
+                );
+              },
+            );
+          }),
+    );
+  }
+  
+  String _getOrganizerFilterString(List<String> organisers, Map<String, bool> showOrganizer) {
+    List<String> show = [];
+    for(String o in organisers) {
```

Comment:
Da würde ich in dart immer mit einem forEach arbeiten ;)

Reply:
Ich eigentlich auch, da beschwert sich mein linter in Android Studio aber immer und das ist der Quick Fix: https://dart.dev/tools/linter-rules/avoid_function_literals_in_foreach_calls